c++/include/orc/Type.hh

```diff
@@ -70,10 +70,11 @@ namespace orc {
     /**
      * Create a row batch for this type.
      */
-    virtual ORC_UNIQUE_PTR<ColumnVectorBatch> createRowBatch(uint64_t size,
-                                                             MemoryPool& pool,
-                                                             bool encoded = false
-                                                             ) const = 0;
+    virtual ORC_UNIQUE_PTR<ColumnVectorBatch> createRowBatch(
+      uint64_t size,
+      MemoryPool& pool,
+      bool encoded = false,
+      bool enableFixedWidthNumericVectorBatch = false) const = 0;
```

Comment:
@luffy-zh When you add one parameter, please avoid any change to the existing parameters. For example, indentation changes on line 73 to 76.

Reply:
In addition, can we keep the original three-parameter method and add new four-parameter method with new parameter additionally?
```
bool enableFixedWidthNumericVectorBatch = false) const = 0;
```